README.md

```diff
@@ -67,28 +67,6 @@ In the response example:
 
 However, since the clientside code is responsible for both rendering and rehydrating from a server-side render, other than the `data` field, the remainder of the response can be whatever the given client usage requires as long as the rendering code and the rehydration code have the same expectations.
 
-## Component Sandboxes
```

Comment:
Is this no longer true?  Perhaps it was already removed?

Reply:
Yeah, every request is in its own VM with no caching - since webpack.